dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipRequestParamsMapper.java

```diff
@@ -76,10 +75,7 @@ public RelationshipOperationParams map(RequestParams requestParams) throws BadRe
             ObjectUtils.firstNonNull(
                     trackedEntity, requestParams.getEnrollment(), requestParams.getEvent())
                 .getValue())
-        .page(requestParams.getPage())
-        .pageSize(requestParams.getPageSize())
-        .totalPages(requestParams.isTotalPages())
-        .skipPaging(toBooleanDefaultIfNull(requestParams.isSkipPaging(), false))
+        .pagingAndSortingCriteriaAdapter(requestParams)
```

Comment:
Are you passing it for simplicity? We are using page, pageSize, ... in the other entities, right?

Reply:
Some here, I am just preparing the params as they are expected by the repository layer.
But maybe here something a little bit better can be done and I could create the adapter again when we are mapping operationParams into queryParams, so I have the part in the controller that is clean and I do not need to change the repository part.
I will look into that